Gopkg.toml

```diff
@@ -33,41 +33,41 @@ required = [
   "k8s.io/code-generator/cmd/deepcopy-gen/args"
 ]
 
+[[constraint]]
+  name = "sigs.k8s.io/controller-runtime"
+  version = "0.1.10"
+
 [[constraint]]
   name = "github.com/portworx/sched-ops"
-  branch = "master"
+  branch = "kubernetes-1.13.1"
 
 [[override]]
   name = "github.com/libopenstorage/stork"
   branch = "master"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.11.0"
+  version = "kubernetes-1.13.1"
 
-[[constraint]]
-  name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.11.0"
-
-[[constraint]]
+[[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.11.0"
+  version = "kubernetes-1.13.1"
 
 [[override]]
-  name = "k8s.io/apiserver"
-  version = "kubernetes-1.11.0"
-
-[[constraint]]
   name = "k8s.io/client-go"
-  version = "8.0.0"
+  version = "kubernetes-1.13.1"
 
-[[constraint]]
+[[override]]
```

Comment:
can we just work with constraints? `override` is more of a force option.

Reply:
This is necessary for now because some of the other packages try to vendor in older version of k8s. Once all the dependent packages are upgraded, we can change it back to constraint.

Reply:
Can you file an issue just so that we don't forget?

Reply:
Done